meson.build

```diff
@@ -85,27 +86,27 @@ link_args = [
 ############################################################
 nitrosdk_dep = dependency('NitroSDK',
     default_options: {
-        'mwrap_ver': '2.0/sp1p2',
+        'mwrap_ver': '3.0-126',
```

Comment:
While I suppose this works, I'd rather update the submodules to support an `mwrap_sdk` option, reflecting the `-wrap:sdk` option. This doesn't just change the compiler, but also the header files used.